e2e/jest.test-setup.ts

```diff
@@ -128,45 +152,69 @@ beforeEach(async () => {
   });
 
   page.on('requestfinished', async (request) => {
-    if (canLogResponse(request)) {
-      if (isApiFailure(request)) {
-        await logError(request);
-      } else {
-        await logResponse(request);
+    let method;
+    let url;
+    let status;
+    try {
+      if (canLogResponse(request)) {
+        // Try find out what the request was if exception thrown.
```

Comment:
This comment seems misleading. I don't see anything about an exception here?

Reply:
right. Updated comment: `Save data for log in catch block when exception is thrown.`